exports.handleGeneralError = (msg, err) => {
    console.log("An error occured: ", { message: msg, error: err});
};